tests/cpp/test_resharding.cpp

```diff
@@ -40,8 +40,7 @@ class ReshardingTest : public NVFuserFixtureParamTest<ReshardingTestParams> {
     // FusionExecutorCache, simplify validation by using
     // FusionExecutorCache::getMostRecentKernelRuntime()->fusionSegments()->groups().
     for (auto expr : fusion_->exprs()) {
-      EXPECT_TRUE(!isResharding(expr) || isLowerableToCommunication(expr))
-          << "on expr=" << expr;
+      EXPECT_TRUE(HostIrLower::canLower(expr)) << "on expr=" << expr;
```

Comment:
```suggestion
      EXPECT_TRUE(HostIrLower::canLower(expr)) << "on expr: " << expr;
```
Nit: it's slightly more readable to use `:` because `expr->toString()` likely contains `=`.